buildspec_combined.yaml

```diff
@@ -0,0 +1,84 @@
+version: 0.1
+component: build
+timeoutInSeconds: 20000
+runAs: root
+shell: bash
+env:
+  exportedVariables:
+   - GITHUB_MD5_HASH
+   - GITLAB_MD5_HASH
+   - BITBUCKET_CLOUD_MD5_HASH
+   - BITBUCKET_SERVER_MD5_HASH
+   - PUBLIC_GITLAB_SERVER_MD5_HASH
+   - VBS_MD5_HASH
+steps:
+  - type: Command
+    name: "Calculating MD5 Hash of Github Large Repo"
+    command: |
+      cd ${OCI_WORKSPACE_DIR}
+      zip -d TestBuildSource_GitHub.zip */.git/*
```

Comment:
I suppose this deletes the `.git` directory from the zip archive. But, where are we creating the archive?

Reply:
The zip files are already present when the repo is pulled
![image](https://user-images.githubusercontent.com/32415315/185887767-18e4bfcd-f67a-4188-aa9e-64ecc73455f6.png)